src/main/kotlin/mate/academy/model/IsoscelesTrapezoid.kt

```diff
@@ -0,0 +1,12 @@
+package mate.academy.model
+
+class IsoscelesTrapezoid (private val baseShort : Double, private val baseLong : Double
+, private val height : Double, color: Color) : Figure(color) {
+    override fun draw() {
+        println("Figure: square, area: ${getArea()}. units" +
```

Comment:
You may use triple quote marks with relevant trim extension function to avoid concatenation.